src/ucp/wireup/wireup_ep.c

```diff
@@ -640,11 +640,11 @@ ucp_wireup_sockaddr_set_tl_lane(ucp_ep_h ucp_ep, uct_ep_h tl_ep,
 ssize_t ucp_wireup_sockaddr_cm_priv_pack_cb(void *arg, const char *dev_name,
                                             void *priv_data)
 {
-    ucp_wireup_client_data_t *client_data = priv_data;
-    ucp_ep_h ep                           = arg;
-    ucp_worker_h worker                   = ep->worker;
-    ucp_ep_config_key_t key               = ucp_ep_config(ep)->key;
-    uct_ep_h tl_ep                        = NULL;
+    ucp_wireup_sockaddr_data_t *client_data = priv_data;
```

Comment:
if this function is called from the server flow then client_data isn't the right name. please rename